app/scripts/controllers/paymentModal.js

```diff
@@ -311,4 +338,22 @@ angular.module('confRegistrationWebApp')
       var registrationPromoCodes = _.pluck($scope.registration.promotions, 'id');
       return !_.contains(registrationPromoCodes, p.id);
     };
+
+    /*
+    Returns false is payment if payment and refund channel are credit card and current time is less than 24 hours from
+    payment time.  If a refund is issued before the payment settles, authorize.net will do a full refund regardless of amount.
+    */
+    $scope.isPartialRefundAvailable = function(payment, refundChannel) {
```

Comment:
This doesn't need to be a $scope function if it's not being accessed by the view.  Change this to var isPartialRefundAvailable = function...